src/main/java/io/stargate/sgv2/jsonapi/api/model/command/Command.java

```diff
@@ -33,6 +36,7 @@
  *
  * <p>Each command should validate itself using the <i>javax.validation</i> framework.
  */
+@RegisterForReflection
```

Comment:
Interestingly enough, without this Native build failed "allowEmptyOptions" tests. Not 100% sure what is missing (I'd have thought this class and subtypes referenced would be reachable) but adding annotation resolved the issue.